src/environment.cpp

```diff
@@ -172,10 +172,12 @@ void Environment::continueRaycast(RaycastState *state, PointedThing *result)
 			bool is_colliding = false;
 			// Minimal distance of all collisions
 			float min_distance_sq = 10000000;
+			// ID of the current box
+			u16 id = 0;
 
 			v3f npf = intToFloat(np, BS);
 			for (std::vector<aabb3f>::const_iterator i = boxes.begin();
-					i != boxes.end(); ++i) {
+					i != boxes.end(); ++i, ++id) {
```

Comment:
continue works on a for range based loop, and id can be incremented in loop, it's just 2 ++id